server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -239,6 +239,7 @@
     }
 
     private PlatformConfig getPlatformConfig(String platform) throws ExtenderException {
+        PlatformConfig commonPlatformConfig = config.platforms.get("common");
```

Comment:
Not ussed?

Reply:
Yeah, I started out with one idea of "merging" these two classes into one (might do later on), but instead implemented the getPlatformConfigProperty() instead for now.